src/sseq.rs

```diff
@@ -417,6 +473,7 @@ mod sseq_test {
             match opt {
                 HammingIterOpt::SkipNBase => 3 * non_n,
                 HammingIterOpt::MutateNBase => 3 * non_n + 4 * n_bases,
+                _ => unreachable!(),
```

Comment:
This is a nit, but I think `unimplemented!()` is probably more appropriate, as there is nothing in this code path that actually makes this unreachable.